### PR TITLE
AI webinar initial draft

### PR DESCRIPTION
**Pull request by knolleary**
## Description

Initial webinar abstract for July.

I've left lots of the header fields blank - will need help setting them properly.

@gdziuba @Hasmin-AC - please update as you see fit.

@Yndira-E will open an art request and link back.
 - https://github.com/FlowFuse/website/issues/2277

### CODE REVIEW

File: src/webinars/2024/bringing-ai-to-nodered.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+hubspot:
+    formId:
+---
+
+**In this webinar, FlowFuse CTO and Node-RED creator Nick O'Leary will show how AI is being brought into the Node-RED world within FlowFuse.**
+
+<!--more-->
+
+There's no denying the place AI has in the software development world. Helping users be more productive and creating richer solutions using the power of AI.
+
```
Comment (Hasmin-AC) [inline src/webinars/2024/bringing-ai-to-nodered.md:18]: Add a short introduction of what FlowFuse Assistant is and what it enables, what you can achieve etc.

File: src/webinars/2024/bringing-ai-to-nodered.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+In this webinar, Nick O'Leary will explore the different ways AI can improve your Node-RED applications - from help create the flows using the new FlowFuse Assistant, to building flows that leverage the AI to its full potential.
```
Comment (Hasmin-AC) [inline src/webinars/2024/bringing-ai-to-nodered.md:19]: Mention the ideal target audience of this webinar..."This webinar is ideal for....who are looking to....

File: src/webinars/2024/bringing-ai-to-nodered.md (hunk 1 above)
Comment (knolleary) [reply src/webinars/2024/bringing-ai-to-nodered.md:18]: Have expanded a bit more on the FlowFuse Assistant.

File: src/webinars/2024/bringing-ai-to-nodered.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+title: Bringing AI to Node-RED with FlowFuse Assistant
+subtitle: Join us for this workshop on infusing Node-RED with AI within FlowFuse.
+image:
+date: 
+time: 
+duration: 60
+hosts: ["nick-oleary"]
+hubspot:
+    formId:
```
Comment (Hasmin-AC) [inline src/webinars/2024/bringing-ai-to-nodered.md:10]:
```suggestion
    formId: febea57b-ef8d-437e-b1be-43f419bff291
```

File: src/webinars/2024/bringing-ai-to-nodered.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+title: Bringing AI to Node-RED with FlowFuse Assistant
+subtitle: Join us for this workshop on infusing Node-RED with AI within FlowFuse.
+image:
+date: 
+time:
```
Comment (Hasmin-AC) [inline src/webinars/2024/bringing-ai-to-nodered.md:6]:
```suggestion
time: 17:00 CET (11:00am ET) 
```

File: src/webinars/2024/bringing-ai-to-nodered.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+---
+title: Bringing AI to Node-RED with FlowFuse Assistant
+subtitle: Join us for this workshop on infusing Node-RED with AI within FlowFuse.
+image:
+date:
```
Comment (Hasmin-AC) [inline src/webinars/2024/bringing-ai-to-nodered.md:5]:
```suggestion
date: 2024-07-31
```

File: src/webinars/2024/bringing-ai-to-nodered.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+---
+title: Bringing AI to Node-RED with FlowFuse Assistant
+subtitle: Join us for this workshop on infusing Node-RED with AI within FlowFuse.
```
Comment (Hasmin-AC) [inline src/webinars/2024/bringing-ai-to-nodered.md:3]:
```suggestion
subtitle: Join us for this live webinar on infusing Node-RED with AI within FlowFuse.
```

**Review by Hasmin-AC — APPROVED** [review]

**Comment by Hasmin-AC** [comment]
@knolleary this page is ready but let me know if you prefer to wait for the release to be done to publish so we are not pre-announcing FlowFuse Assistant

**Pull request merged**